### ISSUE TITLE
Allow multiple pre hook scripts

### ISSUE THREAD

**Issue by tuxmaster5000** [reconquest/atlassian-external-hooks#111]
In many scenarios it will be much simpler to use more simple script instant of one big script.
For example, one for reject binary check in, an second for syntax check etc.
Doing it all in one script will result in much double code, because for example you will need the "binary block code" in every script.

**Comment by kovetskiy** [comment]
Yes, that makes sense, although I'm not sure that it's really technically possible in terms of Bitbucket 
 API extensibility. 

I'll check and get back to you.
